twake/frontend/src/app/components/Leftbar/Channel/Channel.scss

```diff
@@ -378,3 +378,12 @@
     }
   }
 }
+
+.ant-avatar-group .ant-avatar:not(:first-child) {
```

Comment:
Unlike VueJS, css files are not local to the component, so this modification will affect all ant-avatar-group in Twake. Your modification is only of your channel bar, so please put this under something like .channel-bar { .ant-avatar-group .ant-avatar:not(:first-child) { ... } }